### PR TITLE
i18n(zh-cn): update integration docs

### PR DESCRIPTION
**Pull request by 100gle**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

update integration docs since #4796

**Note that** 

1. `guides/integrations-guide/node.mdx` has been reverted, so I just removed unused import content to make it be updated. 
~~2. some updates has synced from #4822, so this PR **should be merged after** #4822~~

#### Related Issue / Implementation PR

- Closes Issue #

### CODE REVIEW

File: src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx [hunk 1]
```diff
@@ -204,6 +291,50 @@ export default defineConfig({
...
+将 `wasmModuleImports: true` 添加到 `astro.config.mjs` 以在 Cloudflare 构建和 Astro 开发服务器中启用该功能。
+
+```diff
+// astro.config.mjs
+import {defineConfig} from "astro/config";
+import cloudflare from '@astrojs/cloudflare';
+
+export default defineConfig({
+    adapter: cloudflare({
++       wasmModuleImports: true
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx:311]: 这个 diff 语法可以改一下

File: src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx [hunk 2]
```diff
@@ -204,6 +291,50 @@ export default defineConfig({
...
+
+启用之后，你可以在 Astro 中通过 `.wasm?module` 语法来导入一个 WebAssembly。
+
+以下是导入一个 Wasm 模块的示例，该模块通过将请求的数字参数相加来返回响应：
+
+```javascript
+// pages/add/[a]/[b].js
+import mod from '../util/add.wasm?module';
+
+// instantiate ahead of time to share module
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx:325]: 这句话需要翻译

File: src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx [hunk 3]
```diff
@@ -204,6 +291,50 @@ export default defineConfig({
...
+const addModule: any = new WebAssembly.Instance(mod);
+
+export async function GET(context) {
+  const a = Number.parseInt(context.params.a);
+  const b = Number.parseInt(context.params.b);
+  return new Response(`${addModule.exports.add(a, b)}`);
+}
+```
+
+虽然这个例子很简单，但是 Wasm 用来加速在不涉及大量 I/O 的情况下的计算密集型操作，比如嵌入图像处理库。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx:335]:
```suggestion
虽然这个例子很简单，但是 Wasm 可以用来加速在不涉及大量 I/O 的情况下的计算密集型操作，比如嵌入图像处理库。
```

**Review by liruifengv — CHANGES_REQUESTED** [review]

File: src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx [hunk 4]
```diff
@@ -72,20 +72,107 @@ export default defineConfig({
...
+    }
+    ```
+
+### `routes.include`
+
+`routes.include: string[]`
+
+默认值：`[]`
+
+如果你想使用自动生成的 `_routes.json`，但又想要包含额外的路由（例如，当你有自定义函数在 `functions` 文件夹中时），那么你可以使用 `routes.include` 选项将额外的路由添加到 `include` 数组中。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx:150]: Suggestion:

```suggestion
如果你想使用自动生成的 `_routes.json`，但又想要包含额外的路由（例如，当你在 `functions` 文件夹中有自定义函数时），那么你可以使用 `routes.include` 选项将额外的路由添加到 `include` 数组中。
```

Reordering sentence.

File: src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx [hunk 5]
```diff
@@ -204,6 +291,50 @@ export default defineConfig({
 });
 ```
 
+## Wasm 模块导入
+
+`wasmModuleImports: boolean`
+
+默认值：`false`
+
+不论是否直接导入 `.wasm` 文件，都会将其视作 ES 模块。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx:300]: Request changes:

```markdown
Whether or not to import `.wasm` files [directly as ES modules](https://github.com/WebAssembly/esm-integration/tree/main/proposals/esm-integration).
```

这句话的意思应该是

```suggestion
是否将 `.wasm` 文件[直接作为 ES 模块](https://github.com/WebAssembly/esm-integration/tree/main/proposals/esm-integration)导入。
```

File: src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx [hunk 6]
```diff
@@ -204,6 +291,50 @@ export default defineConfig({
...
+    adapter: cloudflare({
+      wasmModuleImports: true
+    }),
+	  output: 'server'
+})
+```
+
+启用之后，你可以在 Astro 中通过 `.wasm?module` 语法来导入一个 WebAssembly。
+
+以下是导入一个 Wasm 模块的示例，该模块通过将请求的数字参数相加来返回响应：
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx:319]: Suggestion:

```markdown
The following is an example of importing a Wasm module that then responds to requests by adding the request's number parameters together.
```

```suggestion
以下是导入一个 Wasm 模块的示例，该模块返回了请求的数字参数的和：
```

**Review by Genteure — CHANGES_REQUESTED** [review]

File: src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx [hunk 7]
```diff
@@ -204,6 +291,50 @@ export default defineConfig({
...
+
+```javascript
+// pages/add/[a]/[b].js
+import mod from '../util/add.wasm?module';
+
+// 在使用之前实例化
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx:325]:
```
instantiate ahead of time to share module
```

```suggestion
// 预先初始化以共享同一个模块
```

File: src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx (hunk 5 above)
Comment (100gle) [reply src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx:300]: 按照原文直译的时候，这个表达其实有点怪，你可以细品一下，既不像疑问句又不像陈述句……

Comment (Genteure) [reply src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx:300]: 没有吧？它有点类似“这个设置项控制的是‘是否将文件作为 ES 模块导入’”这种感觉。
设置为 `true` 就是“是”、“启用”、“（肯定性的）把文件作为 ES 模块导入”，设置为 `false` 就是“否”、“禁用”、“不把文件作为 ES 模块导入”。

翻译成“不论是否直接导入 `.wasm` 文件，都会将其视作 ES 模块”，看到这句话之后我的反应是：那这个设置项的作用是什么？

**Review by Genteure — APPROVED** [review]
Content: LGTM 👍
Code block formatting (expressive code instead of `````` ```diff ``````): LGTM, as long as differences between languages is acceptable.

File: src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx [hunk 8]
```diff
@@ -72,20 +72,107 @@ export default defineConfig({
...
+```js ins={8}
+// astro.config.mjs
 import {defineConfig} from "astro/config";
 import cloudflare from '@astrojs/cloudflare';
 
 export default defineConfig({
      adapter: cloudflare({
         mode: 'directory',
-+       functionPerRoute: true
+        functionPerRoute: true
```
Comment (Yan-Thomas) [inline src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx:83]: I recommend continuing to use the normal diff syntax here, I might in the future take care of this conversion automatically within the script and then I'd be fine with having integrations with the more complex Expressive Code cases like this.

Comment (100gle) [reply src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx:83]: ok. Just let me revert them.

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the great work as always peeps! 💜

**Pull request merged**